README.md

```diff
@@ -2,7 +2,9 @@
 
 | Windows Build Status |
 | -------------------- |
-| [![Build status](https://ci.appveyor.com/api/projects/status/9hb88yawy54b0086?svg=true)](https://ci.appveyor.com/project/xqemu-bot/xqemu-manager) |
+| [![Build status](https://ci.appveyor.com/api/projects/status/9hb88yawy54b0086/branch/master?svg=true)](https://ci.appveyor.com/project/xqemu-bot/xqemu-manager?branch=master) |
+| [Latest Build](https://ci.appveyor.com/api/projects/xqemu-bot/xqemu-manager/artifacts/xqemu-manager.zip?branch=master) |
```

Comment:
Two columns here but a 1 column header?

Reply:
Should be a single column, do you think it would look better as a single row?

Reply:
Oops, misread. No, it's fine as a second row. Thanks!